cornflow-server/setup.py

```diff
@@ -0,0 +1,33 @@
+import setuptools
+
+with open("README.rst") as fh:
+    long_description = fh.read()
+
+required = []
+with open("requirements.txt", "r") as fh:
+    required.append(fh.read().splitlines())
+
+setuptools.setup(
+    name="cornflow",
+    version="1.0.0a6",
+    author="baobab soluciones",
+    author_email="sistemas@baobabsoluciones.es",
```

Comment:
I think it's better to contact with cornflow group at cornflow@baobabsoluciones.es